pr_testing/run-pr-comparisons

```diff
@@ -309,7 +309,7 @@ for maxmem in $(find $WORKSPACE/data/PR-${PR_NUM} -follow -name 'maxmem_profile_
 done
 $CMS_BOT_DIR/comparisons/compare-maxmem-summary.py -i $OUTPUT_DIR -f '*.json' -F html -o $OUTPUT_DIR/index.html -u $JENKINS_ARTIFACTS_URL/$PR_BASELINE_DIR || true
 if grep "exceeds threshold" $OUTPUT_DIR/*.err 2>/dev/null; then
-  echo "MAXMEM_COMPARISON${TEST_FLAVOR_STR};OK,max memory used ${UC_TEST_FLAVOR} comparison,See results,/SDT/jenkins-artifacts/$COMP_UPLOAD_DIR/maxmem-comparison" >> ${RESULTS_FILE}
+  echo "FAILED_MAXMEM_COMPARISON${TEST_FLAVOR_STR};OK,max memory used ${UC_TEST_FLAVOR} comparison failed,See failed results,/SDT/jenkins-artifacts/$COMP_UPLOAD_DIR/maxmem-comparison" >> ${RESULTS_FILE}
```

Comment:
Where does this message end up in? Is it
* the summary page, like here https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-aba732/49506/summary.html
* the summary message, like here https://github.com/cms-sw/cmsdist/pull/10199#issuecomment-3544212742

I find the word "failed" confusing here. How about something along "threshold exceeded"? Or does some subsequent script use "failed" as a keyword?

Reply:
I think it ends up in the summary page. I am still trying to figure out if failed needs to be in the message for it to be highlighted.